src/main/java/jimmy/mcgymmy/model/food/exceptions/DuplicateFoodException.java

```diff
@@ -0,0 +1,11 @@
+package jimmy.mcgymmy.model.food.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Food items
+ * (Food items are considered duplicates if they have the same name and nutrition content).
+ */
+public class DuplicateFoodException extends RuntimeException {
```

Comment:
We should extend Exception instead because they are checked

extending `RuntimeException` will lead to a lot of very painful uncaught errors